service-onboarding-build-pack/Jenkinsfile

```diff
@@ -221,44 +223,50 @@ node  {
 
             withCredentials([[$class: 'UsernamePasswordMultiBinding', credentialsId: repo_credential_id, passwordVariable: 'PWD', usernameVariable: 'UNAME']]) {
 				try {
+					events.sendStartedEvent("ADD_WEBHOOK", 'service repo creation starts', context_map)
 					def scm_webhook_target_url
 					if (configLoader.SCM.TYPE == "gitlab") {
 						scm_webhook_target_url = "http://${configLoader.JENKINS.USERNAME}:${configLoader.JENKINS.JENKINS_PASSWORD}@${JenkinsLocationConfiguration.get().getUrl().split('/')[2]}/project/Gitlab-Trigger-Job"
 					} else if (configLoader.SCM.TYPE == "bitbucket") {
 						scm_webhook_target_url = JenkinsLocationConfiguration.get().getUrl() + "/bitbucket-scmsource-hook/notify"
 					}
+					scmModule.setScmEventsWebhookUrl("${g_base_url}/jazz/scm-web-hook")
```

Comment:
Why aren't we calling the `scmModule.addWebhook` with `"${g_base_url}/jazz/scm-web-hook"` as the argument?

Reply:
Fixed